server/tests/batch_comparison/test_batch_tallies.py

```diff
@@ -160,6 +160,55 @@ def test_batch_tallies_replace(
 
     file_id = Jurisdiction.query.get(jurisdiction_ids[0]).batch_tallies_file_id
 
+    rv = client.delete(
+        f"/api/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/batch-tallies",
+    )
+    assert_ok(rv)
+
+    rv = client.get(
+        f"/api/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/batch-tallies"
+    )
+    assert json.loads(rv.data) == {"file": None, "processing": None}
+
+    jurisdiction = Jurisdiction.query.get(jurisdiction_ids[0])
+    assert jurisdiction.batch_tallies_file_id is None
+    assert File.query.get(file_id) is None
+    assert jurisdiction.batch_tallies is None
+
+    set_logged_in_user(client, UserType.AUDIT_ADMIN, DEFAULT_AA_EMAIL)
+    rv = client.get(
+        f"/api/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/batch-tallies/csv"
+    )
+    assert rv.status_code == 404
```

Comment:
Oh wait, actually JAs don't have access to the download link (it's just not part of the JA UI for file uploads currently), so that's why the test is this way. I think they could definitely have access in the future if we upgrade their UI to match the new AA UI, but that's for later